flake.nix

```diff
@@ -107,7 +107,7 @@
     eachDefaultSystemOutputs = flake-utils.lib.eachDefaultSystem (system: let
       pkgs = importNixpkgs system [rust-overlay.overlays.default];
       treefmtEval = loadTreefmt pkgs;
-      toplevel = name: config: nameValuePair "${name}-toplevel" config.config.system.build.toplevel;
+      toplevel = name: config: nameValuePair "nixosTests/${name}" config.config.system.build.toplevel;
```

Comment:
Is this `-toplevel` suffix a useful indication of anything?

Reply:
Yes. That it builds `config.system.build.toplevel` of some `config`, which has the name `name`. The derivation is *not* a NixOS test, as the new name suggests. I am strongly against this change.